packages/react-start/src/plugin/vite.ts

```
@@ -1,15 +1,15 @@
 import {
   START_ENVIRONMENT_NAMES,
   tanStackStartVite,
+  type TanStackStartViteInputConfig,
```

ESLint

[error] Member 'TanStackStartViteInputConfig' of the import declaration should be sorted alphabetically.

(sort-imports)